src/hostapi/coreaudio/pa_mac_core.c

```diff
@@ -1131,31 +1132,31 @@ static OSStatus SetupDevicePropertyListeners( PaMacCoreStream *stream, AudioDevi
+
+    PaMacCore_AudioDeviceAddPropertyListener( deviceID, 0, isInput, kAudioDevicePropertyActualSampleRate,
                                    AudioDevicePropertyActualSampleRateListenerProc, stream );
-    
-    AudioDeviceAddPropertyListener( deviceID, 0, isInput, kAudioStreamPropertyLatency, 
-                                   AudioDevicePropertyGenericListenerProc, stream );
-    AudioDeviceAddPropertyListener( deviceID, 0, isInput, kAudioDevicePropertyBufferFrameSize, 
-                                   AudioDevicePropertyGenericListenerProc, stream );
-    AudioDeviceAddPropertyListener( deviceID, 0, isInput, kAudioDevicePropertySafetyOffset, 
-                                   AudioDevicePropertyGenericListenerProc, stream );
-    
+
+    PaMacCore_AudioDeviceAddPropertyListener( deviceID, 0, isInput, kAudioStreamPropertyLatency,
+                                      AudioDevicePropertyGenericListenerProc, stream );
+    PaMacCore_AudioDeviceAddPropertyListener( deviceID, 0, isInput, kAudioDevicePropertyBufferFrameSize,
+                                      AudioDevicePropertyGenericListenerProc, stream );
+    PaMacCore_AudioDeviceAddPropertyListener( deviceID, 0, isInput, kAudioDevicePropertySafetyOffset,
+                                      AudioDevicePropertyGenericListenerProc, stream );
+
     return osErr;
 }
 
 static void CleanupDevicePropertyListeners( PaMacCoreStream *stream, AudioDeviceID deviceID, Boolean isInput )
-{    
-    AudioDeviceRemovePropertyListener( deviceID, 0, isInput, kAudioDevicePropertyActualSampleRate, 
-                                   AudioDevicePropertyActualSampleRateListenerProc );
-    
-    AudioDeviceRemovePropertyListener( deviceID, 0, isInput, kAudioDevicePropertyLatency, 
-                                   AudioDevicePropertyGenericListenerProc );	
-    AudioDeviceRemovePropertyListener( deviceID, 0, isInput, kAudioDevicePropertyBufferFrameSize, 
-                                   AudioDevicePropertyGenericListenerProc );
-    AudioDeviceRemovePropertyListener( deviceID, 0, isInput, kAudioDevicePropertySafetyOffset, 
-                                   AudioDevicePropertyGenericListenerProc );
+{
+    PaMacCore_AudioDeviceRemovePropertyListener( deviceID, 0, isInput, kAudioDevicePropertyActualSampleRate,
+                                         AudioDevicePropertyActualSampleRateListenerProc, stream );
+
+    PaMacCore_AudioDeviceRemovePropertyListener( deviceID, 0, isInput, kAudioDevicePropertyLatency,
+                                         AudioDevicePropertyGenericListenerProc, stream );
+    PaMacCore_AudioDeviceRemovePropertyListener( deviceID, 0, isInput, kAudioDevicePropertyBufferFrameSize,
+                                         AudioDevicePropertyGenericListenerProc, stream );
+    PaMacCore_AudioDeviceRemovePropertyListener( deviceID, 0, isInput, kAudioDevicePropertySafetyOffset,
+                                         AudioDevicePropertyGenericListenerProc, stream );
```

Comment:
I think you should fix the whitespace here so that it lines up with the `(` like it used to.

Reply:
Done! Thanks!